features/multi_config/test_package/conanfile.py

```diff
@@ -0,0 +1,16 @@
+import os
+from conans import ConanFile, CMake
+
+
+class HelloTestConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def test(self):
+        os.chdir("bin")
+        self.run(".%sexample" % os.sep, run_environment=True)
```

Comment:
Can we check the output here? I know these are not tests, but for the test package itself it would be nice something like:

```python
output = self.run(".%sexample" % os.sep, run_environment=True)
if self.settings.build_type == "Debug":
    assert output == "Hello World Debug!"
else:
    assert output == "Hello World Release!"
```
...just to be sure we are using the proper library.

Reply:
ok, let's do it